static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -379,25 +362,6 @@ export function ReplayCell({
     );
   }
 
-  const subText = (
-    <Cols>
-      <Row gap={1}>
-        <Row gap={0.5}>
```

Comment:
These three things go away. The whole thing was wrapped with `<Row gap={0.5}>{subText}</Row>` before, so now we just keep that one row, and inline everything else.